modules/era/sql/mob_groups_era.sql

```diff
@@ -684,3 +686,5 @@ UPDATE mob_groups SET minLevel='51',maxLevel='54' WHERE name='Balayang' AND grou
 UPDATE mob_groups SET content_tag='WOTG' WHERE name='Mind_Hoarder' AND groupid='11' AND zoneid='204';
 UPDATE mob_groups SET content_tag='WOTG' WHERE name='Sluagh' AND groupid='5' AND zoneid='204';
 UPDATE mob_groups SET content_tag='SYNERGY' WHERE name='Jenglot' AND groupid='7' AND zoneid='204';
+
+UNLOCK TABLES;
```

Comment:
Need new line at end of file here

Reply:
Noted.  Will add tonight!

Reply:
@paladindamarus I already added it within the PR. No worries on this one.